src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/search/report/ExtractionReporter.java

```diff
@@ -57,9 +57,9 @@ public void endedExtraction(final Extraction result) {
         logger.info(ReportConstants.SEPERATOR);
         logger.info("Finished extraction: " + result.result.toString());
         logger.info("Code locations found: " + result.codeLocations.size());
-        if (result.result == ExtractionResult.Exception) {
+        if (result.result == ExtractionResultType.Exception) {
             logger.info("Exception:", result.error);
-        } else if (result.result == ExtractionResult.Failure) {
+        } else if (result.result == ExtractionResultType.Failure) {
             logger.info(result.description);
```

Comment:
Should this log be logger.error not logger.info?

Reply:
Oh yeah, good catch. Updated.